style/buttons.css

```diff
@@ -0,0 +1,25 @@
+.submit-button {
+    background-color: var(--white);
+    font-size: inherit;
+    border: 1px solid var(--black);
+    padding: 0.3em 0.6em;
+    border-radius: 0 0.3rem 0.3rem 0;
+}
+
+.repo-card__details-btn {
+    border: none;
+    background: transparent;
+    color: var(--card-text);
+    cursor: pointer;
+}
+
+.cancel-button {
+    border: none;
+    background: transparent;
+}
+
+.footer-btn {
+    border: none;
+    color: navy;
+    background: none;
+}
```

Comment:
You shouldn't name these buttons based on where they are used, but how they look. Remember what Bootstrap has:


```
btn-primary
btn-outline
...
```

Try and refactor them to 2-3 types of buttons